modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/IgniteTxLocalState.java

```diff
@@ -41,4 +43,12 @@
      *
      */
     public void seal();
+
+    /**
+     * Updates owners threadId to current thread id(used in pessimistic transactions resuming)
+     *
+     * @param oldThreadId owner's initial threadId
```

Comment:
The dot in the end of comment is necessary.
Start form upper case.
@param oldThreadId Owner's initial threadId.
@param uuid Node id.